packages/storefront-lwc/src/modules/commerce/basket/basket.js

```diff
@@ -31,8 +31,8 @@ export default class Basket extends LightningElement {
     }
 
     get shippingMethods() {
-        let shippingMethods =
-            ShoppingBasket.basket.shippingMethods.applicableShippingMethods;
+        let shippingMethods = this.basket.shippingMethods
```

Comment:
clean up: just rely on this.basket as the singleton is going away (in Abhinav's work).